pkg/helper/pipelinerun/description/description.go

```diff
@@ -62,6 +62,12 @@ STARTED	DURATION	STATUS
 {{- end }}
 {{- end }}
 
+{{decorate "timeout" ""}} {{decorate "underline bold" "Timeout\n"}}
+{{ $timeout := .PipelineRun.Spec.Timeout.Duration.String }}{{ if not $timeout }} 
```

Comment:
Agree with @eddycharly that there needs to be a nil check before trying to access `.Duration`. In this case, perhaps the way to go is to display Timeout.Duration if it's present and don't display the Timeout property as part of the output if it's nil.

Reply:
Thanks @eddycharly that's what I was missing out on, that fixed the tests as well.